lib/core/multi_index.cpp

```diff
@@ -126,10 +130,11 @@ MultiIndex<N>::MultiIndex(binned_tag, const Dimensions &inner_dims,
   const Dim slice_dim = get_slice_dim(params.bucketParams()...);
 
   // NOLINTNEXTLINE(cppcoreguidelines-prefer-member-initializer)
-  m_inner_ndim = flatten_dims(
-      make_span(m_stride, 0), make_span(m_shape, 0), inner_dims,
-      inner_dims.index(slice_dim),
-      params.bucketParams() ? params.bucketParams().strides : Strides{}...);
+  m_inner_ndim =
+      flatten_dims(make_span(m_stride, 0), make_span(m_shape, 0), inner_dims,
+                   inner_dims.index(slice_dim),
+                   params.bucketParams() ? params.bucketParams().strides
+                                         : Strides{0, 0, 0, 0, 0, 0}...);
```

Comment:
This requires an allocation now because `NDIM_MAX` > `NDIM_STACK`, right?

Reply:
Good catch!